docs/dev/setup.sh

```diff
@@ -1,34 +1,60 @@
 #!/bin/sh
 
-stream_backend="$1"
-
-if [ -z "$stream_backend" ] || (echo "stan jetstream" | grep -v -q "$stream_backend"); then
-  stream_backend="stan"
-fi
-echo "Using $stream_backend"
-
+OSTYPE=$(uname -s)
 COMPOSE_CMD='docker-compose'
+CLUSTER_NAME="demo-a"
+COMPOSE_FILE="./docs/dev/armada-compose.yaml"
 
-DCLIENT_VERSION=$(docker version -f '{{.Client.Version}}')
-if echo $DCLIENT_VERSION | grep -q '^2' ; then
+docker compose > /dev/null 2>&1
```

Comment:
Why is this here?

Reply:
Because checking for Docker Version 20 or higher is not a reliable heuristic for the existence of "Docker Compose 2" (in which `docker-compose` (written in Python) was replaced by Go code directly as a `docker` sub-command).  So the only way to determine if an installation actually has the preferred new `docker compose` command is to run it, and check the exit status.